src/Interpreters/JIT/compileFunction.cpp

```diff
@@ -94,12 +94,14 @@ static void compileFunction(llvm::Module & module, const IFunctionBase & functio
 
     /// Initialize loop
 
+    auto * end = llvm::BasicBlock::Create(b.getContext(), "end", func);
```

Comment:
Could you please write an explanation of the generated code in comments?
Something like a pseudocode of the generated constructions?

Reply:
Otherwise I just don't understand what's happening :crying_cat_face:

Reply:
Updated.